src/MockBot.Api/Models/Constants.cs

```diff
@@ -8,8 +8,8 @@ public static class Constants
     {
         public const string MessageIdHeaderKey = "X-Message-Id";
         public const string MessageIdRefHeaderKey = "X-Message-Id-Ref";
-        public const string SendToHeaderKey = "X-Sent-By";
-        public const string SentByHeaderKey = "X-Send-To";
+        public const string SendToHeaderKey = "X-Send-To";
```

Comment:
This was a painful discovery :D Can't wait for the common library for these constants.